forge-gui/res/lists/token-images.txt

```diff
@@ -2413,10 +2413,12 @@ copy_voc.jpg https://downloads.cardforge.org/images/tokens/copy_voc.jpg
 copy_vow.jpg https://downloads.cardforge.org/images/tokens/copy_vow.jpg
 copy_woc.jpg https://downloads.cardforge.org/images/tokens/copy_woc.jpg
 copy_znr.jpg https://downloads.cardforge.org/images/tokens/copy_znr.jpg
+cyberman.jpg https://downloads.cardforge.org/images/tokens/cyberman.jpg
 day_mid.jpg https://downloads.cardforge.org/images/tokens/day_mid.jpg
 foretell.jpg https://downloads.cardforge.org/images/tokens/foretell.jpg
 foretell_khm.jpg https://downloads.cardforge.org/images/tokens/foretell_khm.jpg
 hidden.jpg https://downloads.cardforge.org/images/tokens/hidden.jpg
+horror.jpg https://downloads.cardforge.org/images/tokens/horror.jpg
```

Comment:
there is currently no backside for the face down horror by wizards

Reply:
I uploaded that custom image as alternative instead 
![horror](https://github.com/Card-Forge/forge/assets/9781539/8b2b37d3-9fa8-417c-aa35-6891058c89b6)